app/script/view_model/ModalsViewModel.js

```diff
@@ -138,7 +143,7 @@ z.viewModel.ModalsViewModel = class ModalsViewModel {
       this.logger.info(`Show modal of type '${type}'`);
     }
 
-    modal.autoclose = Boolean(options.preventClose);
+    modal.autoclose = !preventClose;
```

Comment:
That's the opposite of what it was doing.. I guess you meant `!!`.. but I still prefer `Boolean` (even though I like the `!!`)

Reply:
Nope. It is exactly what I meant. It was wrong before.